<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bingo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: rgb(40, 40, 40);
	color: rgb(255, 255, 255);
}
QPushButton, QComboBox {
	background-color: rgb(81, 81, 81);
}

QTableView {
	gridline-color: rgb(127, 127, 127);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QTableView" name="bingoBoard">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="sidebar">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QWidget" name="hostSidebar" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="hostSidebarLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="newBoardButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>24</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>120</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>New</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>13</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="listSelector">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>24</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>120</width>
               <height>24</height>
              </size>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QComboBox::SizeAdjustPolicy::AdjustToMinimumContentsLengthWithIcon</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>24</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>120</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>Edit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="newListButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>24</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>120</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>Create new list</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>13</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="removeListButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>Remove list</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>320</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="connectButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>24</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>120</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hostButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>24</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>120</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Start hosting</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>newBoardButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetBoard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newListButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>launchNewEditor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>launchEditor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeListButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeCurrentList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectionToggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hostButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>hostingToggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>584</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetBoard()</slot>
  <slot>launchEditor()</slot>
  <slot>launchNewEditor()</slot>
  <slot>removeCurrentList()</slot>
  <slot>connectionToggle()</slot>
  <slot>hostingToggle()</slot>
 </slots>
</ui>
